### PR TITLE
r2pm: Simplify cleaning code

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

This pr reverts #17357, #17411 and #17442 as per  https://github.com/radareorg/radare2/pull/17357#issuecomment-666164765 and https://github.com/radareorg/radare2/pull/17442#issuecomment-673539719, and to simplify the r2pm cleaning code.

**Test plan**

`r2pm -ci r2dec` / `r2ghidra-dec` / `retdec-r2plugin` still works.

**Closing issues**

...

### CODE REVIEW

File: binr/r2pm/r2pm [hunk 1]
```diff
@@ -597,9 +581,12 @@ case "$1" in
...
 			fi
 
-			if [ "${R2PM_AUTOCLEAN_GITDIR}" = 1 ]; then
-				R2PM_CLEAN_GITDIR "$2"
-				RC=$?
+			if [ -d "${R2PM_GITDIR}/$2" ]; then
+				echo "Cleaning up ${R2PM_GITDIR}/$2..."
+				rm -rf "${R2PM_GITDIR}/$2"
+			else
+				echo "Cannot find $2 in ${R2PM_GITDIR}"
```
Comment (trufae) [inline binr/r2pm/r2pm:588]: > /dev/stderr

Comment (kazarmy) [reply binr/r2pm/r2pm]: ok 187f8d6bf165fd5ecb35bdb46e435f61189fc925

File: binr/r2pm/r2pm [hunk 2]
```diff
@@ -583,23 +568,25 @@ case "$1" in
 	if [ -n "$2" ]; then
 		RC=0
 		while [ -n "$2" ]; do
-			echo "Cleaning $2..."
+			echo "Cleaning $2..." >&2
```
Comment (trufae) [inline binr/r2pm/r2pm:571]: imho this is a status message, not an error/warning and those should be printed to stduot, not stderr

File: binr/r2pm/r2pm [hunk 3]
```diff
@@ -583,23 +568,25 @@ case "$1" in
...
-				echo "Cannot find $FILE"
+				echo "Cannot find $FILE" >&2
 				RC=1
 			fi
 
-			if [ "${R2PM_AUTOCLEAN_GITDIR}" = 1 ]; then
-				R2PM_CLEAN_GITDIR "$2"
-				RC=$?
+			if [ -d "${R2PM_GITDIR}/$2" ]; then
+				echo "Cleaning up ${R2PM_GITDIR}/$2..." >&2
```
Comment (trufae) [inline binr/r2pm/r2pm:585]: same here

Comment (kazarmy) [reply binr/r2pm/r2pm]: ok 6599f70acb4bc888b143af7e888f729690ece654

File: binr/r2pm/r2pm (hunk 2 above)
Comment (kazarmy) [reply binr/r2pm/r2pm]: ok 6599f70acb4bc888b143af7e888f729690ece654

**Pull request merged**